stable/insights-agent/README.md

```diff
@@ -0,0 +1,51 @@
+# Polaris
+
+The Fairwinds Insights Agent is a collection of reporting tools, which send data back
+to [Fairwinds Insights](https://insights.fairwinds.com). If you'd like access
+to Fairwinds Insights, contact `opensource@reactiveops.com`
+
+## Installation
+We recommend installing `insights-agent` in its own namespace.
+
+```
+helm repo add fairwinds https://charts.fairwinds.com/stable
```

Comment:
We generally try to stick to the `fairwinds-stable` convention due to our customers using incubator frequently.

Reply:
Done